packages/core/src/components/chat.ts

```diff
@@ -154,5 +181,16 @@ export function setupChat(room: Room, options?: ChatOptions) {
   }
   room.once(RoomEvent.Disconnected, destroy);
 
-  return { messageObservable: messagesObservable, isSendingObservable: isSending$, send, update };
+  return {
+    messageObservable: messagesObservable,
+    isSendingObservable: isSending$,
+    send,
+    update,
+  };
+}
+
+function isIgnorableChatMessage(
```

Comment:
it might make more sense to filter these in the base SDK instead - or in addition - so devs aren't confused if they see both go by?

Reply:
the base SDK does not have a concept of the legacy chat though. This was all defined within the components framework.

Reply:
if your project uses the new chat protocol, whether it uses components or not, won't it be sending and receiving both types? and the old one goes down the data channel as a user packet so if you listen to user packets on the data channel you'll see them and it might confuse you. Might be better for the base SDK to just drop them before they reach the user's own handlers?

Reply:
> whether it uses components or not, won't it be sending and receiving both types?

it will only send both types if you're using components